incubator/crate/README.md

```diff
@@ -0,0 +1,36 @@
+# CrateDB Helm Chart
+
+Deploy a cluster of CrateDB instances on K8S with this chart.
+
+The `templates` directory contains the definition of the service and the statefulset.
+
+The `values.yml` file contains the default values you can tweak for the `crate.yml` template.
+
+Install this chart from this folder using `helm install .`.
+You may want to adjust first your preferred number of nodes in `values.yml`.
+
+## Configuration
```

Comment:
Is it intentional, that the README does not document all available parameters? If so, would it make sense to keep somewhere a list of all parameters?

Reply:
I think this should be that list, but as discussed elsewhere if there's no reason to be configurable items shouldn't be in that list. When we've settled on a `values.yaml` I'll go through and make sure they're all documented here.